packages/workflow/src/Workflow.ts

```diff
@@ -946,9 +1034,8 @@ export class Workflow {
 			connectionInputData = connectionInputData.slice(0, 1);
 			const newInputData: ITaskDataConnections = {};
 			for (const inputName of Object.keys(inputData)) {
-				newInputData[inputName] = inputData[inputName].map(input => {
-					return input && input.slice(0, 1);
-				});
+				// eslint-disable-next-line @typescript-eslint/prefer-optional-chain
+				newInputData[inputName] = inputData[inputName].map((input) => input && input.slice(0, 1));
```

Comment:
Why does optional chaining does not apply here?

Reply:
because `input && input.slice(0, 1)` is equal `undefined` if input does not exist, but `input?.slice(0,1)` is equal `null`. I would need to adjust the type to accept null, which I was too lazy to do so I disabled the rule 🙈